bin/picca_Pk1D_postprocess.py

```diff
@@ -0,0 +1,251 @@
+                        'if weight-method is fit_snr')
+
+    parser.add_argument('--snr-cut-scheme',
+                        type=str,
+                        default=None,
+                        help='Choice of SNR cut type, '
+                             'Possible options: eboss (varying cut vs z, as in eBOSS - DR14); fixed;' 
+                             'None (obligatory when --weight-method != no_weights)')
+
+    parser.add_argument('--snrcut',
+                        type=float,
+                        default=None,
+                        help='Value of the SNR cut if snr-cut-scheme=fixed')
+
+    parser.add_argument('--overwrite',
+                        action='store_true',
+                        default=False,
+                        required=False,
+                        help='Overwrite the output')
+
+    parser.add_argument('--velunits',
+                        action='store_true',
+                        default=False,
+                        required=False,
+                        help='Compute mean P1D in velocity units')
+
+    parser.add_argument('--no-median',
+                        action='store_true',
+                        default=False,
+                        required=False,
+                        help='Skip computation of median quantities')
+
+    parser.add_argument('--old-output',
+                        action='store_true',
+                        default=False,
+                        required=False,
+                        help='Output the older version of the average P1D')
+
+    parser.add_argument('--ncpu',
+                        type=int,
+                        default=8,
+                        required=False,
+                        help='Number of CPUs used to read input P1D files')
+
+    args = parser.parse_args(sys.argv[1:])
+
+    if (args.weight_method != 'no_weights') and (args.snr_cut_scheme is not None):
+        raise ValueError("""You are using a weighting method with a
+                            redshift-dependent SNR quality cut, this is not
+                            tested and should bias the result""")
+
+    snr_cut_mean = None
+    zbins_snr_cut_mean = None
+    
+    if args.snr_cut_scheme == 'eboss':
+        snr_cut_mean =       [4.1, 3.9, 3.6, 3.2, 2.9, 2.6, 2.2, 2.0, 2.0,
+                              2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0,
+                              2.0, 2.0, 2.0]
+        zbins_snr_cut_mean = np.arange(2.2, 6.4, 0.2)
+    elif args.snr_cut_scheme == 'fixed':
+        snr_cut_mean = args.snrcut
+    elif args.snr_cut_scheme == None:
+        snr_cut_mean = None
+        zbins_snr_cut_mean = None
+    else:
+        raise ValueError("Unknown value for option --snr-cut-scheme"
+                         "You may add here in the code a specific SNR cutting scheme")
+
+    kedge_min, kedge_max, kedge_bin = define_wavenumber_array(args.kedge_min,
```

Comment:
I agree.
This would require some change in the code (cannot pass the full 2D table of k_edges(z) as a simple argument).
=> to be included in another PR, once we settle what we want as a default?